### PR TITLE
F/stats

### PR DESCRIPTION
**Pull request by kduffie**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,6 +39,8 @@
   </style>
   <script src="/s/scripts/@webcomponents-2.1.3b/webcomponentsjs/webcomponents-loader.js"></script>
   <script async src="/dist/guild-nav.js"></script>
+  <script src="/node_modules/moment/min/moment.min.js"></script>
+  <script src="/node_modules/moment-timezone/builds/moment-timezone-with-data-2012-2022.min.js"></script>
 </head>
```
Comment (pshihn) [inline index.html]: You need to make an equivalent change in index-nav.html which is served by the server.

**Comment by pshihn** [comment]
Rest all looks fine

**Comment by kduffie** [comment]
Not sure where to assume we can find those momentjs files when not running
locally.  Do we need to place them in /s?

**Comment by pshihn** [comment]
/s would be a good placeholder. 
I will find a es6 module version later. 

you can probably also do <script async src="{{&public_base}}/node_modules... etc"></script>
(not sure this path will work)

File: index-nav.html [hunk 2]
```diff
@@ -47,6 +47,8 @@
     }
   </style>
   <script src="/s/scripts/@webcomponents-2.1.3b/webcomponentsjs/webcomponents-loader.js"></script>
+  <script src="/s/scripts/moment/min/moment.min.js"></script>
+  <script src="/s/scripts/moment-timezone-with-data.min.js"></script>
```
Comment (pshihn) [inline index-nav.html]: It's not under /min/ based on what you checked in scripts folder

**Comment by kduffie** [comment]
Yeah.  Just fixed that.  Thanks.

**Pull request merged**